lib/solvers/MspConnectionPairSolver/MspConnectionPairSolver.ts

```diff
@@ -70,6 +75,24 @@ export class MspConnectionPairSolver extends BaseSolver {
 
     this.queuedDcNetIds = Object.keys(netConnMap.netMap)
   }
+  private canPinsConnect(
```

Comment:
Shouldnt be a class method. Move to separate file. Should be called checkIfMspPairCanConnectDirectly